<?xml version="1.0"?>
<launch>
  <arg name="base_frame" default="base_link"/>
  <arg name="tool_frame" default="tool0"/>
  <arg name="image_topic" default="camera/color/image_raw"/>
  <arg name="save_dir" default="/tmp/calibration_data"/>
  <arg name="target_file"/>

  <node pkg="rct_ros_tools_rclcpp" exec="command_line_data_collection">
    <param name="base_frame" value="$(var base_frame)"/>
    <param name="tool_frame" value="$(var tool_frame)"/>
    <param name="image_topic" value="$(var image_topic)"/>
    <param name="save_dir" value="$(var save_dir)"/>
    <param name="target_file" value="$(var target_file)"/>
  </node>
</launch>
